_FIPS201/requirements.md

```diff
@@ -165,8 +165,8 @@ PIV enrollment records SHOULD include the following data:
     the name and role of the acquiring agent, the office and organization, time, place, and acquisition
     method. The enrollment data record MAY also document unavailable biometric data or failed attempts
     to collect biometric data. The enrollment data record MAY contain historical biometric data records.
-+ The most recent unique identifiers issued to the individual, such as the Federal Agency Smart Credential Number (FASC-N) and 
-    the card Universally Unique Identifier (UUID). The record MAY contain historical
++ The most recent unique identifiers issued to the individual, such as the Federal Agency Smart Credential Number (FASC-N),
```

Comment:
I am not sure we should have cardholder identifier listed hereThis talks about "most recent" identifier. In the context of card identifiers, "most recent'  makes sense, but not in the context of cardholder identifier.  I suggest revert our resolution to deny.

Reply:
Wouldn't it be possible for the same person, over time, to have different cardholder UUIDs? At least in theory? 

Or would this information be better in a different bullet? As I recall, the goal of the resolution was to add cardholder UUID to this list somewhere, not necessarily in this bullet.